msdk-io/msdk-io-netcdf/src/test/java/io/github/msdk/io/netcdf/NetCDFFileExportMethodTest.java

```diff
@@ -0,0 +1,155 @@
+
+    // Import the file
+    File inputFile = new File(TEST_DATA_PATH + "5peptideFT.mzML");
+    Assert.assertTrue(inputFile.canRead());
+    MzMLFileImportMethod parser = new MzMLFileImportMethod(inputFile);
+    RawDataFile rawFile = parser.execute();
+    Assert.assertNotNull(rawFile);
+    Assert.assertEquals(1.0, parser.getFinishedPercentage(), 0.0001);
+
+    // Export the file to a new mzML
+    File tempFile = File.createTempFile("msdk", ".cdf");
+    tempFile.deleteOnExit();
+    NetCDFFileExportMethod exporter = new NetCDFFileExportMethod(rawFile, tempFile);
+    exporter.execute();
+    Assert.assertEquals(1.0, exporter.getFinishedPercentage(), 0.0001);
+
+    // Import the new cdf file
+    NetCDFFileImportMethod importer = new NetCDFFileImportMethod(tempFile);
+    RawDataFile newRawFile = importer.execute();
+    Assert.assertNotNull(newRawFile);
+    Assert.assertEquals(1.0, importer.getFinishedPercentage(), 0.0001);
+
+    // The file has 7 scans, 2 pass the predicate
+    List<MsScan> scans = newRawFile.getScans();
+    Assert.assertNotNull(scans);
+    Assert.assertEquals(7, scans.size());
+
+    // 2nd scan, #2
+    MsScan scan2 = scans.get(1);
+    Assert.assertEquals(Integer.valueOf(2), scan2.getScanNumber());
+    Assert.assertEquals(MsSpectrumType.PROFILE, scan2.getSpectrumType());
+    Assert.assertEquals(Integer.valueOf(1), scan2.getMsLevel());
+    Assert.assertEquals(0.474f, scan2.getRetentionTime(), 0.01f);
+    // TODO Should find the right place to store polarity in the cdf file
+    // Assert.assertEquals(PolarityType.POSITIVE, scan2.getPolarity());
+    Assert.assertEquals(209.1818184554577, scan2.getMzValues()[100], 0.00001);
+    scan2.getMzValues();
+    intensityBuffer = scan2.getIntensityValues();
+    Assert.assertEquals(19800, (int) scan2.getNumberOfDataPoints());
+    Float scan2maxInt =
+        MsSpectrumUtil.getMaxIntensity(intensityBuffer, scan2.getNumberOfDataPoints());
+    Assert.assertEquals(1.8E5f, scan2maxInt, 1E4f);
+
+    // 5th scan, #5
+    MsScan scan5 = scans.get(4);
+    Assert.assertEquals(Integer.valueOf(5), scan5.getScanNumber());
+    Assert.assertEquals(MsSpectrumType.CENTROIDED, scan5.getSpectrumType());
+    // TODO Should find the right place to store ms level in the cdf file
+    // Assert.assertEquals(Integer.valueOf(2), scan5.getMsLevel());
+    Assert.assertEquals(2.094f, scan5.getRetentionTime(), 0.01f);
+    // TODO Should find the right place to store polarity in the cdf file
+    // Assert.assertEquals(PolarityType.POSITIVE, scan5.getPolarity());
+    Assert.assertEquals(483.4679870605469, scan5.getMzValues()[200], 0.00001);
+    intensityBuffer = scan5.getIntensityValues();
+    Assert.assertEquals(837, (int) scan5.getNumberOfDataPoints());
+    Float scan5maxInt =
+        MsSpectrumUtil.getMaxIntensity(intensityBuffer, scan5.getNumberOfDataPoints());
+    Assert.assertEquals(8.6E3f, scan5maxInt, 1E2f);
+
+    // Cleanup
+    importer.terminate();
```

Comment:
How would access the `NetcdfFile` resource instance of the importer outside the class?